### DISCUSSION TITLE
My Portfolio Website using Flutter

### DISCUSSION THREAD

**Discussion by Tharwatdahroug** [community/community#64889] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

# Tharwat Dahroug 

**Live Preview: 👉🏽 [Tharwat Dahroug](https://tharwatdahroug.web.app)**
## Getting Started

✨ Hello there, I'm Tharwat dahroug. This is my personal website, which contains public information that anyone can find on the internet.

## 🎬 Captured Moment

![cover](https://github.com/Tharwatdahroug/Student_File_C-/assets/56726653/97f5b111-6779-4af0-88d5-f6c71b498080)
## 🌐 Website

<https://github.com/Tharwatdahroug>  
<https://tharwatdahroug.web.app/>  

## 🍃 Branches

Source Code

- [main](https://github.com/Tharwatdahroug/My_portfolio) - Stable and latest version of the source code

## 💙 Built With

- [Flutter](https://flutter.dev)
  - Flutter 3.7.9 • channel stable
  - Tools • Dart 2.19.6 • DevTools 2.20.1
- [GitHub Pages](https://pages.github.com)

## 📌 Prerequisites

- Setup [Flutter](https://flutter.dev) in the development environment.
- Setup a code editor like [Android Studio](https://developer.android.com/studio) or [Visual Studio Code](https://code.visualstudio.com) in the development environment.
- Setup [Git](https://git-scm.com) in the development environment.
- Should be fine with the `flutter doctor` command.
- Anything related to the Flutter app development.

## 💡 How to Setup

- Download or clone the repository.
- Move the project to the selected directory.
- Open it with a code editor such as [Android Studio](https://developer.android.com/studio) or [Visual Studio Code](https://code.visualstudio.com).
- Run `flutter clean` and `pub get` commands.
- **Do not update or upgrade Gradle and other versions until the app is up and running with built versions.**

**Comment by ElielM10100** [comment]
parabéns mano, ficou mt legal

**Reply by Tharwatdahroug** [reply]
thanks brother

**Comment by ggdio** [comment]
Very nice and thanks for sharing the code with the community !

**Comment by ThiagoSchetini** [comment]
nice way to see the flutter working!

**Comment by shujaatsunasra** [comment]
#1 Looks Nice!

**Comment by Shaun289** [comment]
Looks Great!
